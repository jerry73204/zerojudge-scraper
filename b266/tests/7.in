6 8 95
0 0 6 3 8 1
2 9 1 8 2 1
2 8 3 0 1 3
1 1 9 9 1 6
5 2 2 4 2 1
7 3 5 2 5 7
5 1 5 0 3 9
5 4 7 4 6 2
0 1 1 1 1 0 1 1 0 0 0 0 0 0 1 0 0 1 0 0 1 1 0 0 0 1 0 0 0 1 1 0 0 0 0 1 1 1 0 0 0 0 1 1 0 0 1 0 0 1 1 0 1 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 1 0 1 0 1 1 0 1 0 0 0 1 0 0 1 1 1 0 0 1 0 1 0 1 1 1 1
