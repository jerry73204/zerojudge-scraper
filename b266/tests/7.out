6 8
1 1 3 6 1 7 9 2
8 2 1 1 2 5 3 6
3 8 0 9 4 2 0 4
6 1 3 9 2 5 5 7
0 9 8 1 2 3 1 4
0 2 2 1 5 7 5 5
